Makefile

```diff
@@ -166,7 +166,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
 # More info: https://docs.docker.com/develop/develop-images/build_enhancements/
 .PHONY: docker-build
 docker-build: ## Build docker image with the manager.
-	$(CONTAINER_TOOL) build -t ${IMG} .
+	$(CONTAINER_TOOL) buildx build --push --platform "linux/amd64,linux/ppc64le" -t ${IMG} .
```

Comment:
I don't think we need to update this command. We have the `docker-buildx` command below which can be used like:
```
make docker-buildx PLATFORMS='linux/amd64,linux/ppc64le'
```